test/index.js

```diff
@@ -73,6 +79,12 @@ var squareRoot = function(n) {
                : S.Right(Math.sqrt(n));
 };
 
+//  highArity :: a -> b -> c -> d -> a
+var highArity = function(a, b, c, d) { return a; };
+
+//  thunk :: (a -> b) -> c -> a -> b
+var thunk = function(fn) { return function() { return fn; }; };
```

Comment:
I wondered whether it was better no to use `K` within the test suite. Could always use `R.const` if so. Happy to use `K` if no concerns